src/router/index.js

```diff
@@ -3,44 +3,21 @@ import Router from 'vue-router'
 
 import userRoutes from './users'
 
-import Home from '@/views/home/Home'
-import Register from '@/views/register/Register'
-import Login from '@/views/login/Login'
+let utils = require('@/router/utils')
```

Comment:
Let's use imports for everything instead of a mix of requires/imports

```suggestion
import utils from '@/router/utils'
```